LEAF_Request_Portal/sources/Form.php

```diff
@@ -495,9 +520,8 @@ public function getIndicator($indicatorID, $series, $recordID = null, $parseTemp
                 $form[$idx]['value'] = $this->fileToArray($data[0]['data']);
                 $form[$idx]['raw'] = $data[0]['data'];
             }
-
             // special handling for org chart data types
-            if ($data[0]['format'] == 'orgchart_employee'
+            else if ($data[0]['format'] == 'orgchart_employee'
```

Comment:
Would like these to be fed through toLower().  We're working on future-proofing things.

Reply:
These should be left as-is. If the format is coming in with a different case, it should be considered wrong.